### PR TITLE
#68 Use ExpectedException from JUnit 4 in Tests

### PR DESCRIPTION
**Pull request by ChrisA89**
This change is [Reviewable](https://reviewable.io/reviews/google/blockly-android/462)

### CODE REVIEW

File: blocklytest/src/androidTest/java/com/google/blockly/model/FieldNumberTest.java [hunk 1]
```diff
@@ -15,27 +15,38 @@
...
     public void setUp() throws Exception {
-        super.setUp();
         mField = new FieldNumber(FIELD_NAME);
     }
 
+    @Test
     public void testConstructor() {
         assertEquals(FIELD_NAME, mField.getName());
-        assertEquals(Field.TYPE_NUMBER, mField.getType());
+        assertEquals(Field.TYPE_NUMBER, mField.getType(), 0d);
```
Comment (ChrisA89) [inline blocklytest/src/androidTest/java/com/google/blockly/model/FieldNumberTest.java]: The assertEquals that was present was deprecated. The new one takes a precision as well, and was failing tests for me, so I've moved this class to use the new one as part of this change.

**Comment by AnmAtAnm** [comment]
Related to #249.

**Comment by ChrisA89** [comment]
I've got some more commits coming on this. Might be worth holding off reviewing it.

**Comment by AnmAtAnm** [comment]
Awesome stuff. Thanks.  Any chance you're interested in moving more stuff to Junit 4?

**Comment by ChrisA89** [comment]
Yeah I saw #249. I'll have a go at moving some stuff tomorrow. Is there a sensible set of tests for me to start with in case I don't get them all done?

**Pull request merged**